nupic/swarming/hypersearch/utils.py

```diff
@@ -559,3 +560,60 @@ def generatePersistentJobGUID():
 
   """
   return "JOB_UUID1-" + str(uuid.uuid1())
+
+def identityConversion(value, _keys):
```

Comment:
did you mean to delete these new functions from their original location? Or did you intend for them to be duplicates?